src/bootstrap/server-bootstrapper.js

```diff
@@ -60,6 +62,20 @@ export async function createContainer(configValues = {}) {
   logger.info('Registering resource manager');
   container.registerInstance(ResourceManager, resourceManager);
 
+  logger.info('Loading theme files');
+  const globalVariablesFileUrl = new URL(path.resolve('./src/styles/global-variables.less'), import.meta.url);
+  let variables = await lessVariablesToJson(readFileSync(globalVariablesFileUrl, 'utf8'));
+
+  if (serverConfig.themeFile) {
+    const themeFileUrl = new URL(serverConfig.themeFile, import.meta.url);
```

Comment:
Also here there's no need for a URL. `serverConfig.theme` (as all file paths coming from server config) is already an absolute file path and can be used by `readFile(Sync)` as is. The URL construction has no effect, or is there something I don't see?

Reply:
Just copied the same file reading setup from loading `additionalResources`, but I updated both now. And of course reading them async 🤦‍♀️